cmd/cluster/kubevirt/create.go

```diff
@@ -102,5 +96,12 @@ func applyPlatformSpecificsValues(ctx context.Context, exampleOptions *apifixtur
 		RootVolumeStorageClass:    opts.KubevirtPlatform.RootVolumeStorageClass,
 		RootVolumeAccessModes:     opts.KubevirtPlatform.RootVolumeAccessModes,
 	}
+
+	if opts.BaseDomain != "" {
+		exampleOptions.BaseDomain = opts.BaseDomain
+	} else {
+		exampleOptions.Kubevirt.BaseDomainPassthrough = true
```

Comment:
don't we want the default for `BaseDomainPassthrough` boolean property be set in the CRD level?

Reply:
I'm not sure if it's possible to default the value accurately.

The issue is that when BaseDomainPassthrough is set, the hc.Spec.DNS.BaseDomain must be empty so the controller can automatically set it. So, if we default BaseDomainPassthrough=true on the CRD, and the user also set's the DNS.BaseDomain, then the configuration is invalid (unless they set the base domain perfectly to what the passthrough logic would have set it too).

So, we can only default BaseDomanPassthrough when there is no BaseDomain set by the user. I can default this in the CLI, but i don't think i can express that complex default behavior on the CRD today... at least I haven't seen something like that done yet. I've heard the community is improving what can be expressed on the CRD, but i haven't seen it yet.